docs/Modules/overview.md

```diff
@@ -0,0 +1,53 @@
+
+In addition to enabling the module, It also mandates the **moduleSetupData** to setup the module by the smart account during initialisation. It only stores a reference to the module address, thus making it possible for multiple modules to use the same function selector.
+
+Each account must have a default validation module which gets used when no is enabled. All validation modules can act as active validation modules, but the ones with following functionality can act as default validattion module.
+
+## Validation Modules
+![validationModule.png](../images/modules/validationModule.png)
+
+Validation Module is a module that extends abstract contract **BaseAuthorizationModule** - which implements **IBaseAuthorizationModule** interface. This interface requires the implementation of the following methods:
+
+- `validateUserOp`: this method validates userOperation. It expects userOp callData to execute method calls of the Smart account and userOp signature being the ABI-encoded signature and module address.
+- `isValidSignature`: this method validates an EIP-1271 signature.
+- `isValidSignatureUnsafe`: this method validates an EIP-1271 signature but expects the data Hash to already include smart account address information.
+
+Validation modules are invoked with a “call”. It has it’s own storage and doesn’t share storage with Biconomy Smart Account. Since validation module storage is accessed during the validation phase, ERC-4337 [storage rules](https://github.com/eth-infinitism/account-abstraction/blob/abff2aca61a8f0934e533d0d352978055fddbd96/eip/EIPS/eip-4337.md#storage-associated-with-an-address) apply to this.
+
+Following is the list of Validation Modules:
+
+- [**ECDSA Ownership Module**](ecdsa.mdx): This module is widely adopted as a validation module for Biconomy smart accounts. It seamlessly integrates with MPC providers such as Web3Auth, abstracting EOA Private Key storage and enabling a web2-like experience for eg. email login.
+- [**Multichain ECDSA Validator Module**](multichain.mdx): This module significantly improves UX for deploying and setting up Smart Accounts on several chains. It is an extension of ECDSA Module enabling used to dispatch multiple userOps on different chains using single signature.
+- [**Session key Manager Validation Module**](sessionvalidationmodule.mdx): This module enables the use of sessions to execute  transactions. It verifies whether a given user operation adheres to the permissions defined within the session key and confirms that the operation has been signed by that session key. This can only be used as an active validation module.
```

Comment:
```suggestion
- [**Session key Manager Validation Module**](sessionvalidationmodule.mdx): This module enables the use of sessions to execute transactions. It verifies whether a given user operation adheres to the permissions defined within the session key and confirms that the operation has been signed by that session key. This can only be used as an active validation module.
```